### ISSUE TITLE
Add ability to add images directly from links.

### ISSUE THREAD

**Issue by THElegend5** [bpepple/metron#140]
It would be nice to be able to add images directly from image links instead of only being able to add them from your desktop.

**Comment by bpepple** [comment]
Off the top of my head, I'm not sure how you'd do this cleanly without making the primary use case (local upload) sort of a mess. Do you have any examples of sites that implement this feature you requested?

**Comment by THElegend5** [comment]
> 

Yeah comicvine has this feature they give a pop up asking for any possible way you might want to add an image.
![image](https://user-images.githubusercontent.com/63903377/195835788-0c413c4a-f1c2-4c10-81c5-8cb61de0ed38.png)

**Comment by bpepple** [comment]
Interesting... I'll have to go to their site and check it out. Thx.
